### DISCUSSION TITLE
magick-x86 can not find gswin64c version 10.0.0

### DISCUSSION THREAD

**Discussion by mrbod** [ImageMagick/ImageMagick#5618] [Development] [answered]
On windows 11 x64:
magick-x86 can not find gswin64c version 10.0.0 and falls back on gswin32c.exe that does not exist.

With gswin64c of version 9.xx this works. Is there an easy fix for this other than using gs 9.xx or magick x64?

**Comment by snibgo** [comment]
What version of IM are you using? What does "magick -version" say?

You seem to be using IM for x86 which suggests you have a 32-bit processor (very old). So why do you want to use GS for 64-bit processors?

**Comment by dlemstra** [comment, marked as answer]
It looks like they store the value in the registry different than what we are expecting. I just pushed a patch to fix this issue. This will be resolved in next release.

**Reply by mrbod** [reply]
Excellent!
